Formula/gauge.rb

```diff
@@ -42,6 +42,12 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
```

Comment:
@ilovezfs thanks for the clarification. Is there any way we can let our brew users know that Gauge collects telemetry?